### PR TITLE
Error display toy action

### PR DESCRIPTION
**Pull request by reza-ghasemzadeh**

### CODE REVIEW

**Review by mackeynichols — CHANGES_REQUESTED** [review]
Hey @reza-ghasemzadeh, this is good 👍 

I would like to truly merge some of your changes into main :) However, could you please remove the `error_display` action?

I knwo it was just a "toy" action. However, if I want to merge the rest of your changes into main, I'll first need you to remove this "toy" :)

**Comment by reza-ghasemzadeh** [comment]
> Hey @reza-ghasemzadeh, this is good 👍
> 
> I would like to truly merge some of your changes into main :) However, could you please remove the `error_display` action?
> 
> I knwo it was just a "toy" action. However, if I want to merge the rest of your changes into main, I'll first need you to remove this "toy" :)

Hi @mackeynichols, Thanks for the review. I will modify it.

File: ckanext/datastore_profiler/plugin.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ def get_actions(self):
         return {
             "update_profile": datastore_profiler.update_profile,
             "datastore_create": datastore_profiler.datastore_create_hook,
+            "error_display": datastore_profiler.error_display,
```
Comment (mackeynichols) [inline ckanext/datastore_profiler/plugin.py:36]: Thanks for the initial change @reza-ghasemzadeh . However, we'll need to remove this line too. Not only is it misleading to have an action that doesnt exist, but Im worried CKAN might not start up with the extension like this

**Review by mackeynichols — CHANGES_REQUESTED** [review]

File: ckanext/datastore_profiler/plugin.py (hunk 1 above)
Comment (reza-ghasemzadeh) [reply ckanext/datastore_profiler/plugin.py:36]: I will remove it, too. Thanks for the note

**Review by mackeynichols — APPROVED** [review]
Great stuff! I'm approving this to be merged into the main branch

**Pull request merged**